lib/app/modules/bulk_data/bulk_data_export_sequence.rb

```diff
@@ -214,34 +251,83 @@ def check_cancel_request
         check_file_request
       end
 
-      test 'Server should return "202 Accepted" for delete export content' do
+      test 'Server shall return "202 Accepted" for delete export request' do
         metadata do
           id '08'
           link 'https://build.fhir.org/ig/HL7/bulk-data/export/index.html#bulk-data-delete-request'
-          desc %(
+          description %(
+
```

Comment:
The tests all need descriptions to avoid ui issues:
![Screen Shot 2019-10-17 at 8 06 28 AM](https://user-images.githubusercontent.com/15969967/67007382-40e4e400-f0b5-11e9-80c6-ff2b6db6fd97.png)
vs
![Screen Shot 2019-10-17 at 8 06 37 AM](https://user-images.githubusercontent.com/15969967/67007390-44786b00-f0b5-11e9-8b96-d6ac95d83c5c.png)